<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>part inventory manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>761</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Inventory Details</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>581</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>8</height>
       </size>
      </property>
      <property name="text">
       <string>search of referance with count level lower of equal to :-</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>110</y>
        <width>741</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 9pt &quot;Century Gothic&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Referance</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>part name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>min area</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>max area</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nbr of holes</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>min diameter</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>max diameter</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>count</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="search_btn">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>50</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="count_filter_txt">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>19</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Inventory statistics</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of referece :-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of part type :-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>min Number of holes :-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>70</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>max Number of holes :-</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ref_nbr">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 85, 255);
border-color: rgb(0, 85, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_part_nbr">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 85, 255);
border-color: rgb(0, 85, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_min_hole">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>20</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 85, 255);
border-color: rgb(0, 85, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_max_holes">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>70</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 85, 255);
border-color: rgb(0, 85, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTableWidget" name="table2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>531</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Reference</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Part name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>count</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>711</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Top 3 reference with minimum iventory level</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="check_btn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>200</y>
        <width>181</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
    color: silver;
    background-color: #302F2F;
    border-width: 2px;
    border-color: #4A4949;
    border-style: solid;
    padding-top: 2px;
    padding-bottom: 2px;
    padding-left: 10px;
    padding-right: 10px;
    border-radius: 4px;
    /* outline: none; */
    /* min-width: 40px; */
}

QPushButton:disabled
{
    background-color: #302F2F;
    border-width: 2px;
    border-color: #3A3939;
    border-style: solid;
    padding-top: 2px;
    padding-bottom: 2px;
    padding-left: 10px;
    padding-right: 10px;
    /*border-radius: 2px;*/
    color: #808080;
}

QPushButton:focus {
    background-color: #3d8ec9;
    color: white;
}



QPushButton:checked{
    background-color: #4A4949;
    border-color: #6A6969;
}

QPushButton:hover {
    border: 2px solid #78879b;
    color: silver;
}
</string>
      </property>
      <property name="text">
       <string>Check</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_min_hole_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>30</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>REF</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_max_holes_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>80</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>REF</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Edit invertory</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Reference</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Part name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Min Area</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>160</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Max Diameter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>130</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>MIn Diameter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>100</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Holes</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Max Area</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>190</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Count</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ID :-</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="reference">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 85, 255);
border-color: rgb(0, 85, 255);</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="part_name">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 85, 255);
border-color: rgb(0, 85, 255);</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="min_area">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 85, 255);
border-color: rgb(0, 85, 255);</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="max_area">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 85, 255);
border-color: rgb(0, 85, 255);</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="number_of_holes">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 85, 255);
border-color: rgb(0, 85, 255);</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="min_diameter">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 85, 255);
border-color: rgb(0, 85, 255);</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="max_diameter">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>160</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 85, 255);
border-color: rgb(0, 85, 255);</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="next_btn">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>270</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="last_btn">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>270</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="previous_btn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>270</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="first_btn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="update_btn">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_btn">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>270</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Item's</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_btn">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>270</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Item's</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="count">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>190</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 85, 255);
border-color: rgb(0, 85, 255);</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="enter_btn">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>ENTER</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 85, 255);
border-color: rgb(0, 85, 255);</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="tittle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 26pt &quot;Century Gothic&quot;;
color: rgb(176, 107, 255);
background-color: rgb(193, 193, 193);</string>
    </property>
    <property name="text">
     <string>Part inventory manager</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Century Gothic&quot;;QPushButton
{
    color: silver;
    background-color: #302F2F;
    border-width: 2px;
    border-color: #4A4949;
    border-style: solid;
    padding-top: 2px;
    padding-bottom: 2px;
    padding-left: 10px;
    padding-right: 10px;
    border-radius: 4px;
    /* outline: none; */
    /* min-width: 40px; */
}

QPushButton:disabled
{
    background-color: #302F2F;
    border-width: 2px;
    border-color: #3A3939;
    border-style: solid;
    padding-top: 2px;
    padding-bottom: 2px;
    padding-left: 10px;
    padding-right: 10px;
    /*border-radius: 2px;*/
    color: #808080;
}

QPushButton:focus {
    background-color: #3d8ec9;
    color: white;
}



QPushButton:checked{
    background-color: #4A4949;
    border-color: #6A6969;
}

QPushButton:hover {
    border: 2px solid #78879b;
    color: silver;
}
</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>530</y>
      <width>571</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
